config.toml.example

```diff
@@ -384,7 +382,9 @@ changelog-seen = 2
 
 # Sets the number of codegen units to build the standard library with,
 # regardless of what the codegen-unit setting for the rest of the compiler is.
-#codegen-units-std = 1
+# NOTE: building with anything other than 1 is known to occasionally have bugs.
+# See https://github.com/rust-lang/rust/issues/83600.
+#codegen-units-std = codegen-units
```

Comment:
Here `codegen-units-std = codegen-units` use `codegen-units` as default value, that defined as `#codegen-units = if incremental { 256 } else { 16 }`, so it will be not 1 and have some bugs by default(by comment)?

Reply:
Yes, that's correct. It's unclear to me how common the bug is or why it's related to codegen units.